itou/templates/gps/my_groups.html

```diff
@@ -35,39 +35,111 @@ <h3 class="h4 mb-0" id="results">
+                                                    {% if counter > 1 %}
+                                                        sont
+                                                    {% else %}
+                                                        est
+                                                    {% endif %}
+                                                    intervenu{{ counter|pluralizefr }} auprès de ce bénéficiaire.
                                                 {% endif %}
                                             </div>
                                         {% endwith %}
                                     </div>
-                                    <div>
-                                        <a href="{% url 'users:details' public_id=membership.follow_up_group.beneficiary.public_id %}"
-                                           class="btn btn-outline-primary btn-block w-100 w-md-auto"
-                                           aria-label="Consulter la fiche de {{ membership.follow_up_group.beneficiary.get_full_name }}">Consulter la fiche</a>
-                                    </div>
+
+
+                                </div>
+                                <div class="d-flex justify-content-between">
+
+                                    {% with membership.follow_up_group.beneficiary.public_id as public_id %}
+                                        <div>
+                                            <a href="{% url 'users:details' public_id=public_id %}"
+                                               class="btn btn-warning btn-block w-100 w-md-auto btn-ico"
+                                               aria-label="Consulter la fiche de {{ membership.follow_up_group.beneficiary.get_full_name }}"
+                                               data-bs-toggle="modal"
+                                               data-bs-target="#confirm_unfollow_{{ public_id }}">
+                                                <i class="ri-user-unfollow-line" aria-hidden="true"></i>
+                                                <span>Ne plus suivre</span>
+
+                                            </a>
+                                            {% url 'gps:leave_group' group_id=membership.follow_up_group.id as leave_group_url %}
+
+                                            {# To avoid javascript, we create a modal per line. Otherwise we could implement https://getbootstrap.com/docs/5.0/components/modal/#varying-modal-content #}
+                                            {% include "gps/includes/confirm_modal.html" with title="Êtes-vous sûr de ne plus vouloir suivre "|add:membership.follow_up_group.beneficiary.get_full_name|add:" ?" id="confirm_unfollow_"|addstr:public_id body="Vous pourrez toujours suivre ce bénéficiaire de nouveau plus tard." confirm_text="Ne plus suivre" confirm_url=leave_group_url confirm_class="btn-danger" %}
```

Comment:
```suggestion
                                            {% include "gps/includes/confirm_modal.html" with title="Êtes-vous sûr de ne plus vouloir suivre "|add:membership.follow_up_group.beneficiary.get_full_name|add:" ?" id="confirm_unfollow_"|add:public_id body="Vous pourrez toujours suivre ce bénéficiaire de nouveau plus tard." confirm_text="Ne plus suivre" confirm_url=leave_group_url confirm_class="btn-danger" %}
```

Reply:
Non, le souci du add par défaut c'est qu'il essaie de caster les deux valeurs en entier avant de les additionner ce qui fait que ça donne des résultats étonnants en fonction des types qu'on essaie d'additionner. Comme le dit la doc https://docs.djangoproject.com/en/dev/ref/templates/builtins/#add

> This filter will first try to coerce both values to integers. If this fails, it’ll attempt to add the values together anyway. This will work on some data types (strings, list, etc.) and fail on others. If it fails, the result will be an empty string.

Et comme public_id est un UUID, le cast échoue et il finit en chaîne vide et le add ne fonctionne pas, d'où l'ajout de ce addstr, cf https://stackoverflow.com/questions/4386168/how-to-concatenate-strings-in-django-templates

Reply:
Donc ce sera pour une autre PR je suppose ?

Reply:
OK !